pkg/recipes/terraform/execute.go

```diff
@@ -68,7 +68,7 @@ type executor struct {
 func (e *executor) Deploy(ctx context.Context, options Options) (*tfjson.State, error) {
 	// Install Terraform
 	i := install.NewInstaller()
-	tf, err := Install(ctx, i, options.RootDir)
+	tf, err := Install(ctx, i, options.RootDir, options.LogLevel)
```

Comment:
I believe that we should create something like InstallOptions struct and have RootDir and LogLevel properties of that struct. With this approach, we are going to be able to add any properties we want without needing to update the signature of the functions all the time!

Reply:
I think that this is the right time to add that capability.